packages/framework/src/Model/Product/Elasticsearch/ProductIndex.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Product\Elasticsearch;
+
+use Shopsys\FrameworkBundle\Component\Elasticsearch\AbstractIndex;
+
+class ProductIndex extends AbstractIndex
+{
+    public const INDEX_NAME = 'product';
```

Comment:
as the getter exists, is it necessary to have the constant public?

Reply:
Make sense.